### ISSUE TITLE
5.1 does not connect

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Robyer/robyer#8]
```
What steps will reproduce the problem?
1. Version 5.0 worked fine.
2. Updater updated the addon to the version 5.1
3. The status connecting is displayed. No error are reported and no contacts 
are shown.

Win XP SP3
Miranda 0.9.32
```

Original issue reported on code.google.com by `cim...@gmail.com` on 31 Oct 2011 at 11:42

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 5.2
```

Original comment by `ROBYER` on 31 Oct 2011 at 12:33
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, it works.
```

Original comment by `cim...@gmail.com` on 31 Oct 2011 at 12:40
